array_methods_lab.js

```diff
@@ -12,9 +12,13 @@ Output: [11,12,13,14,15]
 
 \n----------------------------------------\n\n`);
 
+let qar1 = [1, 2, 3, 4, 5]
+let ans1 = qar1.map(i => i + 10)
+console.log(ans1);
+
```

Comment:
Great thought process. However, can you rewrite this creating your response as a function.

Reply:
Great!